app/_locales/en/messages.json

```diff
@@ -4268,7 +4268,7 @@
     "message": "Stake"
   },
   "stakeDescription": {
-    "message": "Hold up your crypto and earn potential profits"
+    "message": "Stake your Crypto to earn rewards"
```

Comment:
Is "Crypto" intentionally using title case here? cc @coreyjanssen

Reply:
Nice catch! Updated here https://github.com/MetaMask/metamask-extension/pull/20755/commits/24410e4726d0bb19b5af26ce90a56d13ccd3fe07